src/others/file_explorer/lv_file_explorer.c

```diff
@@ -512,8 +531,8 @@ static void browser_file_event_handler(lv_event_t * e)
             show_dir(obj, (char *)file_name);
```

Comment:
My concern here is that if the directory fails to open for some reason we will end up with a broken path. It is a good idea to revert the path to its previous state instead of assuming that the selected item is not a directory if it failed to open as a directory.

Reply:
@C47D is it already addressed?

Reply:
It will be addressed in a separated PR, see https://github.com/lvgl/lvgl/pull/7124#issuecomment-2430415434